modules/apps/template/template-test/src/testIntegration/java/com/liferay/template/info/item/provider/test/TemplateInfoItemFieldSetProviderTest.java

```diff
@@ -219,10 +221,13 @@ public void testGetInfoFieldSetByClassNameWhenTemplateEntryExists()
 				categoryTemplateEntry.getTemplateEntryId(),
 			infoField.getName());
 
-		Optional<Boolean> optional = infoField.getAttributeOptional(
-			TextInfoFieldType.HTML);
+		Boolean value = (Boolean)infoField.getAttribute(TextInfoFieldType.HTML);
+
+		if (value == null) {
```

Comment:
@JanisZhang Could you review all `InfoField` related changes and see if you can simplify such logic using `GetterUtil` or else?

Reply:
Hi @dantewang, I added 1 commit https://github.com/liferay-core-infra/liferay-portal/pull/2892/commits/fca7a07204282f1b4b2ce4dbe2453ad88b79410c to simplify such logic. Thanks!